pkg/rancher-desktop/backend/lima.ts

```diff
@@ -671,7 +671,14 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
   }
 
   protected static get limactl() {
-    return path.join(paths.resources, os.platform(), 'lima', 'bin', 'limactl');
+    let limactlBin = 'limactl'
+    // Darwin versions for Ventura start at 22.1.0
+    // https://en.wikipedia.org/wiki/MacOS_Ventura#Release_history
+    if (os.platform() == 'darwin' && semver.gte(os.release(), '22.1.0')) {
+      limactlBin = 'limactl-ventura'
```

Comment:
> If someone changes `limactlCodename` in `dependencies.yaml` and doesn't update this value RD will try to execute a binary that does not exist.

I'm tempted to say the the `limactlCodename` should not be included in `dependencies.yaml`, whose main purpose is to alert us to new releases of our dependencies.

I would keep things hard-coded in just the code (but maybe using constants), and not make it configurable via the YAML file at all.

I suspect if we ever need to change anything, the change will be more complex than what we can anticipate right now. So let's wait until that happens, and do the refactoring then, but keep it otherwise simple for now.

Reply:
Sounds good I can do that.

Reply:
Done.